src/write.c

```diff
@@ -387,29 +387,27 @@ avifResult avifEncoderAddImage(avifEncoder * encoder, const avifImage * image, u
         if (image->exif.size > 0) {
             // Validate Exif payload (if any) and find TIFF header offset
             uint32_t exifTiffHeaderOffset = 0;
-            if (image->exif.size > 0) {
```

Comment:
I removed this redundant image->exif.size > 0 check. That's the only change in this area.